flink-dist/src/main/flink-bin/LICENSE

```diff
@@ -218,35 +218,66 @@ licenses.
 The Apache Flink project bundles the following components
 under the Apache License (v 2.0):
 
- - Apache Commons Logging (http://commons.apache.org/proper/commons-logging/)
- - Apache Commons Codec (http://commons.apache.org/proper/commons-codec/)
- - Apache Commons Collections (http://commons.apache.org/collections/)
- - Apache Commons CLI (http://commons.apache.org/cli/)
- - Apache Commons FileUpload (http://commons.apache.org/fileupload/)
- - Apache Commons IO (http://commons.apache.org/io/)
- - Apache Commons Lang 3 v.3.3.2 (http://commons.apache.org/proper/commons-lang/)
- - Apache Commons Math (http://commons.apache.org/proper/commons-math/)
- - Apache log4j 1.2 (http://logging.apache.org/log4j/1.2/)
- - Apache Avro v1.7.6 (http://avro.apache.org)
+ - Apache Commons Logging (commons-logging:commons-logging:1.1.1 - http://commons.apache.org/proper/commons-logging/)
+ - Apache Commons Codec (commons-codec:commons-codec:1.3 - http://commons.apache.org/proper/commons-codec/)
+ - Apache Commons Collections (commons-collections:commons-collections:3.2.1 - http://commons.apache.org/collections/)
+ - Apache Commons CLI (commons-cli:commons-cli:1.2 - http://commons.apache.org/cli/)
+ - Apache Commons FileUpload (commons-fileupload:commons-fileupload:1.3.1 - http://commons.apache.org/fileupload/)
+ - Apache Commons IO (commons-io:commons-io:2.4 - http://commons.apache.org/io/)
+ - Apache Commons Lang 3 (org.apache.commons:commons-lang3:3.3.2 - http://commons.apache.org/proper/commons-lang/)
+ - Apache Commons Math (org.apache.commons:commons-math:2.2 - http://commons.apache.org/proper/commons-math/)
+ - Apache Commons Compress (org.apache.commons:commons-compress:1.4.1 - http://commons.apache.org/proper/commons-compress/)
+ - Apache log4j (log4j:log4j:1.2.17 - http://logging.apache.org/log4j/1.2/)
+ - Apache Avro (org.apache.avro:avro:1.7.6 - http://avro.apache.org)
  - Apache Hadoop (http://hadoop.apache.org)
```

Comment:
Technically, we don't include Hadoop any more in our binary distribution. The reason is, because it is contained in `org.apache.flink:flink-shaded-include-yarn` which contains the shaded Hadoop dependencies. Thus, what we actually have to include here is the `org.apache.flink:flink-shaded-include-yarn` dependency and add its NOTICE file to the NOTICE file of the binary distribution.